build.rs

```diff
@@ -1,3 +1,19 @@
+/// This file is configured using env vars to build ouch with shell completions.
+///
+/// # Examples:
+///
+/// Building ouch and completions (need to be set to `'1'`):
+/// ```sh
+/// GEN_COMPLETIONS=1 cargo build
+/// ```
+///
+/// Building ouch and completions, inside another folder:
+///
+/// ```sh
+/// GEN_COMPLETIONS=1 OUCH_COMPLETIONS_DIR=ouch-completions cargo build
+/// ```
```

Comment:
While doing this, one question came to mind, what if instead of two env vars, I just check for `GEN_COMPLETIONS`, if it's value is `1` or `true`, the code uses `OUT_DIR`, if it's another value, we use as the destination folder.

So usage would be:

```sh
# simple build completions 
GEN_COMPLETIONS=1 cargo build
```

```sh
# to new folder 
GEN_COMPLETIONS=ouch-completions cargo build
```

Waiting for your feedback on this, thanks in advance!

Reply:
Sounds good to me, yeah. Probably the most elegant way to handle this.